crates/core/component/auction/src/auction/dutch/actions/withdraw.rs

```diff
@@ -15,6 +17,44 @@ pub struct ActionDutchAuctionWithdraw {
     pub reserves_commitment: balance::Commitment,
 }
 
+impl ActionDutchAuctionWithdraw {
+    /// Compute a balance **commitment** for this action.
+    ///
+    /// # Diagram
+    ///
+    /// The value balance commitment is built from the balance:
+    ///  ┌────────────────────┬──────────────────────┐
+    ///  │      Burn (-)      │       Mint (+)       │
+    ///  ├────────────────────┼──────────────────────┤
+    ///  │    auction nft     │       auction        │
+    ///  │   with seq >= 1    │    value balance     │
+    ///  └────────────────────┼──────────────────────┤
+    ///                       │withdrawn auction nft │
+    ///                       │      with seq+1      │
+    ///                       └──────────────────────┘
+    ///
+    /// More context: [Actions and Value balance][protocol-spec]
+    /// [protocol-spec]: https://protocol.penumbra.zone/main/transactions.html#actions-and-value-balance
+    pub fn balance_commitment(&self) -> balance::Commitment {
+        let prev_auction_nft = Balance::from(Value {
+            amount: 1u128.into(),
+            // The sequence number should always be >= 1, because we can
+            // only withdraw an auction that has ended (i.e. with sequence number `>=1`).
+            // We use a saturating operation defensively so that we don't underflow.
+            asset_id: AuctionNft::new(self.auction_id, self.seq.saturating_sub(1)).asset_id(),
```

Comment:
My rationale to do this is that we check the binding signature before we run any action validation checks, so even if this MUST be `seq >= 1`, we can't actually rely on that yet.